core/dom/element.js

```diff
@@ -480,17 +480,63 @@ CKEDITOR.dom.element.clearMarkers = function( database, element, removeFromDatab
 		 * Retrieve the bounding rectangle of the current element, in pixels,
 		 * relative to the upper-left corner of the browser's client area.
 		 *
-		 * @returns {Object} The dimensions of the DOM element including
-		 * `left`, `top`, `right`, `bottom`, `width` and `height`.
-		 */
-		getClientRect: function() {
+		 * You can pass additional parameter if the function should give absolute element position which can be used for positioning elements
+		 * inside scrollable areas.
+		 *
+		 * E.g. you could use this function with {@link CKEDITOR.dom.window#getFrame editor's window frame} to
+		 * calculate the absolute rectangle of the visible area of the editor's viewport.
+		 * The retrieved absolute rectangle could be used to position elements like toolbars or notifications (elements outside an editor)
+		 * to always keep them inside an editor viewport independently from the scroll position.
+		 *
+		 * ```javascript
+		 * var frame = editor.window.getFrame();
+		 * frame.getClientRect( true );
+		 * ```
+		 *
+		 * @param {Boolean} [isAbsolute=false] The function will retrieve absolute rectangle of the currect element i.e. including scroll position.
+		 * @returns {Object.<String, Number>} The dimensions of the DOM element.
+		 * @returns {Number} return.top Top offset.
+		 * @returns {Number} return.bottom Bottom offset.
+		 * @returns {Number} return.left Left offset.
+		 * @returns {Number} return.right Right offset.
+		 * @returns {Number} return.height Element height.
+		 * @returns {Number} return.width Element width.
```

Comment:
Please add information about returned `x` and `y` properties (as they're even used in test).